providers/defltprov.c

```diff
@@ -476,6 +476,10 @@ static const OSSL_ALGORITHM deflt_asym_kem[] = {
     { PROV_NAMES_X448, "provider=default", ossl_ecx_asym_kem_functions },
 # endif
     { PROV_NAMES_EC, "provider=default", ossl_ec_asym_kem_functions },
+# ifndef OPENSSL_NO_ECX
+    /* Temporary until PQC hybrids exist */
+    { "X25519-X448", "provider=default", ossl_X25519_X448_kem_functions },
```

Comment:
Should hybrid be in the name?

Reply:
If it were going to stay in.  It's never getting merged.

Reply:
For a transient algorithm that is not going to be merged, I don't see the need :grin:

Reply:
Please note that naming according to https://www.ietf.org/archive/id/draft-kwiatkowski-tls-ecdhe-mlkem-02.html is defined to be X25519MLKEM768 and SecP256r1MLKEM768. If I understand the hybrid algorithm instantiation logic right, it depends on a lexical separation of name components which seems to be not guaranteed. Also, please note that ML-KEM-768 already is in the feature branch by now and could also be used for testing instead of this originally proposed artificial hybrid.

Reply:
Ahh, scrap that comment: Just saw that this code also depends on some constants not yet in the feature branch's includes.

Reply:
Yeah, I went to add the real hybrid algs earlier today and discovered that they'd not made it in yet :)

The naming isn't restricted here.  The sub-algorithms must be listed in the order they appear in the concatenated output but this bears no relation to the overall algorithm name ordering etc.